associations/has_many_association_test.go

```diff
@@ -5,26 +5,27 @@ import (
 	"testing"
 
 	"github.com/gobuffalo/pop/associations"
-	"github.com/gobuffalo/uuid"
+	"github.com/gobuffalo/pop/nulls"
 	"github.com/stretchr/testify/require"
 )
 
-type fooHasMany struct {
-	ID           uuid.UUID    `db:"id"`
-	BarHasManies barHasManies `has_many:"bar_has_manies"`
+type FooHasMany struct {
+	ID           int           `db:"id"`
+	BarHasManies *barHasManies `has_many:"bar_has_manies"`
 }
 
 type barHasMany struct {
-	FooHasManyID uuid.UUID `db:"foo_has_many_id"`
+	Title        string    `db:"title"`
+	FooHasManyID nulls.Int `db:"foo_has_many_id"`
 }
 
 type barHasManies []barHasMany
 
 func Test_Has_Many_Association(t *testing.T) {
 	a := require.New(t)
 
-	id, _ := uuid.NewV1()
-	foo := fooHasMany{ID: id}
+	id := 1
```

Comment:
Why?

Reply:
Hi @robbyoconnor,  thanks for taking a time to review this PR, to answer your question, the change from `uuid` to `int` type is for making sure that it works for integers type as well. I was testing with int and left it like this. I should have created another test for that.